Comparison problems after #286

After #286, I noticed a few changes / errors

1. Whenever a rule file is loaded, it no longer logs the current jruby library version
2. This code now caused an error:
```
logger.info("XX") if Number1 == Number2
```
where Number1 = Number2 = NULL

The error message was:
```
08:31:28.068 [ERROR] [y.OpenHAB::DSL::Rules::AutomationRule] - undefined method `>' for true:TrueClass (NoMethodError)
In rule: x
/openhab/conf/automation/jsr223/ruby/personal/test.rb:44:in `block in <main>'
/openhab/conf/automation/jsr223/ruby/personal/test.rb:40:in `<main>'
```


3. `say` caused an error inside a rule when triggered by a changed trigger, but works with `on_start`:
```
rule 'x' do
  on_start
  run do
    say 'on_start' # This works
  end
end

rule 'FR: triggered ' do
  changed gCameraMotions.members, to: ON
  run do |_item|
    say 'changed' # this caused an error
  end
end
```

Error message:
```
08:28:33.990 [ERROR] [y.OpenHAB::DSL::Rules::AutomationRule] - class or module required (TypeError)
In rule: FR: triggered
/openhab/conf/automation/jsr223/ruby/personal/test.rb:55:in `block in <main>'
```

--- a/features/actions.feature
+++ b/features/actions.feature
@@ -58,3 +58,20 @@ Feature:  actions
     When I deploy the rule
     Then It should log "Action method sendMail exists" within 5 seconds
 
+  Scenario: Use say action wrapper from a rule
+    Given items:
+      | type   | name        | label  | state  |
+      | Switch | LightSwitch | Switch | OFF    |
+    And a rule
+      """
+      rule 'Use say' do
+        changed LightSwitch
+        run do
+          say("something")
+          logger.info("Switch changed")
+        end
+      end
+      """
+    When I deploy the rule
+    And item "LightSwitch" state is changed to "ON"
+    Then It should log "Switch changed" within 5 seconds

--- a/lib/openhab/dsl/actions.rb
+++ b/lib/openhab/dsl/actions.rb
@@ -9,7 +9,6 @@ module OpenHAB
     # Module to import and streamline access to OpenHAB actions
     #
     module Actions
-      java_import org.openhab.core.library.types.PercentType
       include OpenHAB::Log
 
       OpenHAB::Core::OSGI.services('org.openhab.core.model.script.engine.action.ActionService')&.each do |service|
@@ -83,7 +82,7 @@ module OpenHAB
       # @return [void]
       #
       def say(text, voice: nil, sink: nil, volume: nil)
-        volume = PercentType.new(volume&.to_i) unless volume.is_a?(PercentType) || volume.nil?
+        volume = Types::PercentType.new(volume) unless volume.is_a?(Types::PercentType) || volume.nil?
         Voice.say text, voice, sink, volume
       end
 
@@ -97,7 +96,7 @@ module OpenHAB
       # @return [void]
       #
       def play_sound(filename, sink: nil, volume: nil)
-        volume = PercentType.new(volume&.to_i) unless volume.is_a?(PercentType) || volume.nil?
+        volume = Types::PercentType.new(volume) unless volume.is_a?(Types::PercentType) || volume.nil?
         Audio.playSound sink, filename, volume
       end
     end

